packages/frontend-shared/src/components/ShikiHighlight.vue

```diff
@@ -73,29 +73,36 @@ shikiWrapperClasses computed property.
 </template>
 
 <script lang="ts">
-import type { Highlighter } from 'shiki'
-import { getHighlighter, setOnigasmWASM, setCDN } from 'shiki'
+import type { Highlighter, ILanguageRegistration } from 'shiki'
+import { getHighlighter, setOnigasmWASM } from 'shiki'
 import onigasm from 'onigasm/lib/onigasm.wasm?url'
+import shikiCyTheme from '../public/shiki/themes/cypress.theme.json'
+const langTmsRaw = import.meta.globEager('../public/shiki/languages/*.tmLanguage.json')
+
+const langTms: ILanguageRegistration[] = Object.values(langTmsRaw).map((grammar: any) => {
+  return {
+    grammar,
+    id: grammar.name,
+    scopeName: grammar.scopeName,
+  }
+})
 
 setOnigasmWASM(onigasm)
-setCDN(`${import.meta.env.BASE_URL}shiki/`)
 
 let highlighter: Highlighter
 
-export const langsSupported = ['typescript', 'javascript', 'ts', 'js', 'css', 'jsx', 'tsx', 'json', 'yaml', 'html'] as const
+export type CyLangType = 'typescript' | 'javascript' | 'ts' | 'js' | 'css' | 'jsx' | 'tsx' | 'json' | 'yaml' | 'html' | 'plaintext' | 'txt' | 'text' | 'vue' | string
 
-let langs = langsSupported.concat([])
-
-export type CyLangType = typeof langsSupported[number] | 'plaintext' | 'txt'| 'text'
+export const langsSupported = langTms.map((lang: ILanguageRegistration) => lang.id)
 
 export async function initHighlighter () {
   if (highlighter) {
     return
   }
 
   highlighter = await getHighlighter({
-    themes: ['cypress.theme'],
-    langs,
+    theme: shikiCyTheme as any,
+    langs: langTms,
```

Comment:
What does `Tms` stand for in `langTms`?

Reply:
If it's no biggie, I'd prefer to keep the variable as `langs` so there's fewer variables to keep in mind (before we just had `langs`, now we have `langTms` which is then assigned to `langs`)